sw/airborne/boards/parrot_minidrone/mt9v117.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) Freek van Tienen <freek.v.tienen@gmail.com>
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+ /**
+ * @file boards/parrot_minidrone/mt9v117.h
+ *
+ * Initialization and configuration of the MT9V117 CMOS Chip
+ */
+
+#ifndef MT9V117_H
```

Comment:
is this exactly the same sensor as on the bebop? if so duplicating is not ideal as there are updates to the bepop cam that may be nice here. I wouldn't say it is required but you should add it to your list of to do in the issues.